content/material_and_questions.md

```diff
@@ -44,4 +53,153 @@ In the below table, any minimum or maximum variation between material years are
+
+Question Type Maximums
+: For a given Question Type, the maximum number of questions must not be exceeded in all "non-Overtime" questions. This includes any questions appended with "A" or "B."
+
+Overtime Specifics
+Question Type maximums can be exceeded in Overtime.
+
+## Question Writing
+
+### Preparation of Questions
+All questions must contain verbatim material. On some types, the addition of a single, approved, interrogative word at the very beginning or very end is required.
+
+### Question Types
+
+#### Interrogative
+Should not be overly long and should be clear and precise.
+Must have exactly one inserted interrogative word.
+Must have a single answer.
+Must include a unique word, or 2-3 word unique phrase within the first 5 words of the question. 
+The Question Type abbreviation is "INT"
+
+#### Multiple Answer
+Must have exactly one inserted interrogative word.
+Must have more than one answer.
+- The clarification of a single answer does not meet the above requirement.
+- The identification of a singular pronoun does not meet the above requirement.
+- The identification of a plural pronoun does meet the above requirement.
+Must include a unique word, or 2-3 word unique phrase within the first 5 words of the question. 
+If the multiple answers contain at least one "positive" answer AND at least one "negative" answer, this is deemed invalid as a Multiple Answer question.
+The Question Type abbreviation is "MA"
+
+#### Reference
+Used to distinguish exact same words or phrases from the material.
+The entire question is part of the required question + answer.
+If a quizzer needs to provide the reference question, the provided reference question must: 
+- Be of the exact same type as the one on the card (CR, CVR, CRMA, CVRMA).  The quizmaster and/or answer judge must check to confirm this.
+- Have content AND wording is similar to the content and wording of the written question.
+    - For greater clarity, we have provided the following example: CVR According to Luke chapter 23: 37 King of what?  A. The Jews The quizzer can give the question “King of what?” or “King of whom?”, as Jews could be referring to either nation (what) or people (whom).  In this case the quizzer could also give “King of the what?”
+- Have the determining word or phrase from the quizmaster’s card. 
+    - In the 50/50 example, “What men?”, and “Lawless what?, the questions are not similar.
+The determination of a valid Reference question, and the valid sub-type only considers the content in the question. 
+
+##### Chapter-Verse
+Must have a single answer.
+The question must contain a word or phrase that exists more than once in the same chapter.
+Context is the single verse.
+The Question Type abbreviation is "CVR"
+
+##### Multiple-Answer, Chapter-Verse
+Must have more than one answer.
+The Question Type abbreviation is "MACVR"
+
+##### Chapter (only)
+The question must contain a word or phrase that exists more than once in the material.
```

Comment:
Definitely needs an addition, "exists more than once in the material, but only once in a chapter."

Reply:
Okay, this brings up another question. To me, the definitions of INT, CVR, CR when taken together, cover all possible scenarios. So I don't think it's necessary to say under:

INT - cannot contain words/phrases occurring more than once in a verse/chapter
CR - cannot contain words/phrases occurring more than once in a chapter
CVR - cannot contain words/phrases occurring either once in the material or once in the chapter in question

I am defining what the question must contain to be valid.

Not what it can't contain else be invalid.

Thoughts?

Reply:
I think defining both what makes it valid and invalid is useful.

Reply:
Isn't it cleaner to define validity--so then anything else is invalid?

Once you start listing out invalid use cases that are just subsets of the whole, I think it's far more confusing.

Reply:
I'm not advocating only defining invalid use cases. I think defining validity should always come first, but in some cases adding suffix invalidity statements can prevent misinterpretation even though logically they could (and arguably should) be inferred.

Reply:
you're right, but I don't want you to be. Defining validity SHOULD be enough.

Reply:
Well, you could explicitly state at a higher hierarchy level that if not explicitly defined as valid, it's invalid. I could support that, I think.

Reply:
I think that would be a massive change. As I've stated somewhere, I think some sections of the rule book are clearly written to "define things as valid, everything else is invalid" (e.g. QM prompts), while different sections of the rule book are written to "define things as invalid, everything else is valid" (e.g. fouls), while still other sections fall somewhere in the middle (e.g. looking at Question Type requirements AND Invalid Questions combined)

Reply:
What if you explicitly define in super-sections which is which? That would then at least make it explicit, and you wouldn't have to do a ton of rewriting or suffer a lot of content duplication. Right?

Reply:
Yes, I think that works. It works great in the first two examples I mentioned (QM prompts and Fouls). It works...less well..in the 3rd example, where it's more difficult to blanketly state up front what you're stating.